frontend/app/src/components/form/object-form.tsx

```diff
@@ -35,7 +35,7 @@ interface ObjectFormProps extends Omit<DynamicFormProps, "fields"> {
   kind: string;
   onSuccess?: (newObject: any) => void;
   currentObject?: Record<string, AttributeType>;
-  currentProfile?: Record<string, Pick<AttributeType, "value" | "__typename">>;
+  currentProfiles?: IProfileSchema[];
```

Comment:
Why did we change from profite's data to profile's schema typing ?

Reply:
it's part of a previous attempt and it has not been reverted, will update that